server/controllers/user.js

```diff
@@ -0,0 +1,17 @@
+const { User } = require("../database");
+
+module.exports = {
+  assignThread: async (userId, threadId) => {
+    const user = await User.findById(userId);
+    user.assigned_threads.addToSet(threadId);
```

Comment:
one random question, should we be using camelcase or snakecase for fields in the database? I'd probably look that up and see what the convention is

Reply:
Looks like for mongo it's camelcase since it's JS based. Probably should've been consistent with that from the beginning. Updated to follow that convention.